VideoLocker/res/layout/discussion_responses_response_row.xml

```diff
@@ -35,8 +35,6 @@
                 android:textSize="@dimen/edx_x_small"
                 tools:text="This is a response" />
 
-            <include layout="@layout/discussion_author_layout" />
```

Comment:
If `discussion_author_layout` is no longer being used, then it should be deleted, I suppose.